src/lib/interfaces.ts

```diff
@@ -61,4 +61,21 @@ export interface PageContent {
   content: string
   frontmatter: Frontmatter
   tocItems: Array<ToCItem>
+  lastUpdatedDate?: string
+}
+
+export interface RequiredFrontmatter {
+  title: string
+  lang: Lang
+}
+
+export interface UpgradeFrontmatter extends RequiredFrontmatter {
+  description?: string
```

Comment:
nitpicking: looking into the Upgrades layout, seems like the `description` should be required
```suggestion
  description: string
```

Reply:
Yes, it is... I'll patch here, but note this is also patched in the Front matter PR that focuses on all of this in more detail.

In the end, `description` may be a blank string in some files, but the field is _required_ in **all** layouts, along with `title` and `lang`... but yeah, see #42 =)